### PR TITLE
reference/performance/follower-read: refine some terms

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

### What is changed, added or deleted? (Required)

Remove some terms like "Linearizability" and "ReadIndex" from the overview.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

N/A

### CODE REVIEW

File: reference/performance/follower-read.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ category: reference
 
 ## 概述
 
-Follower Read 功能是指在强一致性读的前提下使用 Region 的 follower 副本来承载数据读取的任务，从而提升 TiDB 集群的吞吐能力并降低 leader 负载。Follower Read 包含一系列将 TiKV 读取负载从 Region 的 leader 副本上 offload 到 follower 副本的负载均衡机制。TiKV 的 Follower Read 可以保证数据读取的线性一致性，配合 TiDB Snapshot Isolation 事务隔离级别，可以为用户提供强一致的数据读取能力。
+Follower Read 功能是指在强一致性读的前提下使用 Region 的 follower 副本来承载数据读取的任务，从而提升 TiDB 集群的吞吐能力并降低 leader 负载。Follower Read 包含一系列将 TiKV 读取负载从 Region 的 leader 副本上 offload 到 follower 副本的负载均衡机制。TiKV 的 Follower Read 可以保证数据读取的一致性，配合 TiDB Snapshot Isolation 事务隔离级别，可以为用户提供强一致的数据读取能力。
 
 > **注意：**
 >
-> 为了获得强一致读取的能力，在当前的实现中，follower 节点需要额外付出 `ReadIndex` 开销，因此目前 Follower Read 的主要益处是隔离集群的读写请求以及提升整体读取吞吐。从单个请求的 latency 角度看，会比传统的 leader 读取多付出一次与 Raft `ReadIndex` 的交互开销。
+> 为了获得强一致读取的能力，在当前的实现中，follower 节点需要向 leader 节点询问当前的执行进度 (我们将其称为 `ReadIndex`) ，这会花费一次额外的网络请求开销，因此目前 Follower Read 的主要益处是隔离集群的读写请求以及提升整体读取吞吐。
```
Comment (TomShawn) [inline reference/performance/follower-read.md]:
```suggestion
> 为了获得强一致读取的能力，在当前的实现中，follower 节点需要向 leader 节点询问当前的执行进度（即 `ReadIndex`），这会产生一次额外的网络请求开销，因此目前 Follower Read 的主要优势是处理隔离集群的读写请求以及提升整体读取吞吐。
```

Comment (zz-jason) [reply reference/performance/follower-read.md]: @5kbpers PTAL

**Comment by ChristianLeow** [comment]
@yikeke: I'd like to translate this PR.

**Comment by TomShawn** [comment]
@5kbpers Please address the comment.
@zz-jason PTAL

**Comment by yikeke** [comment]
> @yikeke: I'd like to translate this PR.

Sure. I changed the label and assigned you as the translator. You can start your translation now! (Note that this PR may have further changes until it's merged, but I think the changes will be small so it's okay that you make the corresponding adjustment later in your pull request.)

File: reference/performance/follower-read.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ category: reference
 
 ## 概述
 
-Follower Read 功能是指在强一致性读的前提下使用 Region 的 follower 副本来承载数据读取的任务，从而提升 TiDB 集群的吞吐能力并降低 leader 负载。Follower Read 包含一系列将 TiKV 读取负载从 Region 的 leader 副本上 offload 到 follower 副本的负载均衡机制。TiKV 的 Follower Read 可以保证数据读取的线性一致性，配合 TiDB Snapshot Isolation 事务隔离级别，可以为用户提供强一致的数据读取能力。
+Follower Read 功能是指在强一致性读的前提下使用 Region 的 follower 副本来承载数据读取的任务，从而提升 TiDB 集群的吞吐能力并降低 leader 负载。Follower Read 包含一系列将 TiKV 读取负载从 Region 的 leader 副本上 offload 到 follower 副本的负载均衡机制。TiKV 的 Follower Read 可以保证数据读取的一致性，配合 TiDB Snapshot Isolation 事务隔离级别，可以为用户提供强一致的数据读取能力。
```
Comment (gengliqi) [inline reference/performance/follower-read.md:13]: `强一致性读` should be removed.
`可以保证数据读取的一致性` is ambiguous.
Maybe just `TiKV 的 Follower Read 配合 TiDB Snapshot Isolation 事务隔离级别...`

**Pull request merged**